lib/app/migrate.ts

```diff
@@ -359,11 +378,22 @@ async function beginMigration(
   );
   const { migrationId } = data;
 
-  const pollResponse = await pollMigrationStatus(
-    derivedAccountId,
-    migrationId,
-    [MIGRATION_STATUS.INPUT_REQUIRED]
-  );
+  let pollResponse: MigrationStatus;
+  try {
+    pollResponse = await pollMigrationStatus(derivedAccountId, migrationId, [
+      MIGRATION_STATUS.INPUT_REQUIRED,
+    ]);
+  } catch (error) {
+    SpinniesManager.fail('beginningMigration', {
+      text: lib.migrate.spinners.unableToStartMigration,
+    });
+    if (isMigrationStatus(error) && error.status === MIGRATION_STATUS.FAILURE) {
```

Comment:
Not a blocker, but we do have `isSpecifiedError` in LDL that you could use here (doesn't buy you a ton tbh, outside of maybe consistency in util usage)

Reply:
Good question.  These aren't actually `Error` objects, our polling util rejects with plain objects that look like:
```
{
   status: "SUCCESS" | "FAILURE" | "SOME_OTHER_STATUS"
   ... other fields
}
```

Wheres `isSpecifiedError` only operates on `HubSpotHttpErrors`